x-pack/legacy/plugins/ml/public/data_frame_analytics/common/analytics.ts

```diff
@@ -152,3 +168,62 @@ export const useRefreshAnalyticsList = (
     },
   };
 };
+
+const DEFAULT_SIG_FIGS = 3;
+
+export function getValuesFromResponse(response: RegressionEvaluateResponse) {
+  let meanSquaredError = idx(response, _ => _.regression.mean_squared_error.error) as number;
+  if (meanSquaredError) {
+    meanSquaredError = Number(meanSquaredError.toPrecision(DEFAULT_SIG_FIGS));
+  }
+
+  let rSquared = idx(response, _ => _.regression.r_squared.value) as number;
+  if (rSquared) {
+    rSquared = Number(rSquared.toPrecision(DEFAULT_SIG_FIGS));
+  }
+
+  return { meanSquaredError, rSquared };
+}
+
+export const loadEvalData = async ({
```

Comment:
This should only be loaded once the job is complete

Reply:
Fixed here: 412146a3960c2bd5f532da98ed6102a053d37728